test/utils/shippable/integration.sh

```diff
@@ -2,17 +2,20 @@
 
 source_root=$(python -c "from os import path; print(path.abspath(path.join(path.dirname('$0'), '../../..')))")
 
-test_image="${IMAGE}"
+default_test_image="docker.io/ansible/ansible:centos7"
+default_test_target="all"
+test_image="${IMAGE:-$default_test_image}"
 test_privileged="${PRIVILEGED:-false}"
 test_flags="${TEST_FLAGS:-}"
-test_target="${TARGET:-}"
+test_target="${TARGET:-$default_test_target}"
 test_ansible_dir="${TEST_ANSIBLE_DIR:-/root/ansible}"
 test_python3="${PYTHON3:-}"
 
 http_image="${HTTP_IMAGE:-ansible/ansible:httptester}"
 
 keep_containers="${KEEP_CONTAINERS:-}"
 copy_source="${COPY_SOURCE:-}"
+share_source_dir="${SHARE_SOURCE:-}"
```

Comment:
You'll need to add `copy_source=` in the Shippable conditional below this line: https://github.com/ansible/ansible/pull/17709/files#diff-c578b0d6c6c5833aeb54b3ff282557a1L23